src/helper/Reply.cc

```diff
@@ -192,6 +192,20 @@ Helper::Reply::emptyBuf() const
     return empty;
 }
 
+void
+Helper::checkForUnsupportedAnnotations(const NotePairs &notes, const char *context)
+{
+    const auto entries = notes.expandListEntries(nullptr);
+    for (const auto &e: entries) {
+        const auto &name = e->name();
+        if (!name.length())
```

Comment:
Is it possible to assert that condition or do we really allow nameless annotations? It feels odd to work so hard on classifying and warning about annotation names but then silently allow empty ones.